packages/atlas/src/views/studio/VideoWorkspace/NftForm/AcceptTerms/AcceptTerms.tsx

```diff
@@ -38,17 +37,14 @@ export const AcceptTerms: React.FC<AcceptTermsProps> = ({
   termsAccepted,
   toggleTermsAccept,
 }) => {
-  const { getTotalDaysAndHoursText } = useNftForm()
-  const { convertDurationToBlocks } = useBlockTimeEstimation()
+  const { getNumberOfBlocksAndDaysLeft } = useNftForm()
 
-  const startDate = formData.startDate as Date
-  const endDate = formData.endDate as Date
+  const { startDate, endDate } = formData
 
-  const validDate = isValid(formData.startDate) && isValid(formData.endDate)
+  const isStartDateValid = startDate instanceof Date && isDateValid(startDate)
```

Comment:
Why do we need `instanceof` here? Doesn't `isValid` do the job comprehensively? Isn't this something that isValid should take care of?

Reply:
If a user chooses `1 day/3 days/5 days` as the expiration date in the select dropdown, the `endDate` will be set to a `number`.  If you pass primitive number to `isValid` it will return true - I want to avoid that. 

Anyway, I'll use another utility function from `date-fns` - `isDate` which should take care of date validation here and in the other functions.

Reply:
ok, changed my mind. I'll use just `instanceof Date` everywhere. If I start using `isDate` I'll have to add `as Date` to every valid date, which is annoying.